pootle/apps/pootle_app/forms.py

```diff
@@ -64,13 +64,14 @@ def __init__(self, *args, **kwargs):
             ).count
 
             if has_stores:
-                #self.fields['localfiletype'].widget.attrs['disabled'] = True
+                # self.fields['localfiletype'].widget.attrs['disabled'] = True
```

Comment:
I don't see compelling reasons to keep commented out code — we use VCSs after all, and if there's something important to warn other developers, commented out code without any extra details is not the best approach IMHO.